### DISCUSSION TITLE
Domain size and boundary types for wind simulation

### DISCUSSION THREAD

**Discussion by togersen** [firemodels/fds#13757] [Q&A]
I need to look at the influence of wind on the openings for replacement air, and I have some questions that I hope you can help answer. I will use a model with a constant wind speed and wind direction.

I will adopt an approach with a constant wind speed and direction, specifically using the method referred to as Wind Method 1 in the FDS User Guide. This is to avoid complicating things by, for example, using a logarithmic wind profile or considering terrain roughness.

However, I am somewhat unsure about how high the domain should be relative to the building height, as well as the width of the domain. Additionally, I am uncertain about the upstream and downstream distances in relation to my building’s placement.

Furthermore, how should the boundary conditions for the sides of the calculation domain be handled? Should there be openings on the sides, inflow upstream, an opening downstream, as well as at the top, or should some be free slip?"

Thanks in advance

**Comment by drjfloyd** [comment]
Your side and top conditions should be open so that air can flow in or out.

This paper has some suggestions on domain size: https://link.springer.com/article/10.1007/s10694-018-0749-4

**Comment by mcgratta** [comment]
Set the lateral and top boundaries to `OPEN`. For a simple case, just set the `SPEED` on the `WIND` line. Add horizontal and vertical slices of `VELOCITY` with `VECTOR=T` on the `SLCF` line. Look at the flow vectors near the boundaries to ensure that there is nothing unphysical.

**Comment by togersen** [comment]
Thanks for the replies.

With the reference given above I have made som test with different cell size. The velocity and pressure seems to converges with around 10 cm to 20 cm cells. But if I look at the y+ value it's far away from 30 - 100. So is anyone familiar with some articles that have investigated what cell size is needed for y+ to be 30 - 100 or whether it has no significance overall?
